RESOURCES.md

```diff
@@ -1,27 +1,144 @@
+| <a href="https://www.virtualgis.io/">VirtualGIS<br/><img src="figures/ecosystem/virtualgis.jpg"/></a> | A set of 2D, 3D and Augmented Reality GIS applications for visualizing data based on 3D Tiles and CesiumJS |
+| <a href="https://github.com/iTowns/itowns">iTowns<br/><img src="figures/ecosystem/itowns.jpg"/></a> | A Three.js-based framework for visualizing 3D geospatial data, including 3D Tiles |
+| <a href="https://geopi.pe/">geopipe<br/><img src="figures/ecosystem/geopipe.jpg"/></a> | A platform for digital twins for games, simulation, architecture, with options for efficient visualization in the browser, based on 3D Tiles |
+| <a href="https://data.grandlyon.com/en/">data.grandlyon.com<br/><img src="figures/ecosystem/poulain.jpg"/></a> | A 3D digital clone of Métropole de Lyon, based on 3D Tiles (Cesium blog post: [3D Digital Territory Lab: A Digital Clone of Métropole de Lyon](https://cesium.com/blog/2018/02/05/digital-territory-lab/)) |
+
+
+
+
+### Featured Demos
+
+<sub>(Note: These demos had been published based on the <a href="https://github.com/CesiumGS/cesium/blob/main/CHANGES.md#1871---2021-11-09" target="_blank">CesiumJS 1.87.1 Release</a>, which included experimental support for the extensions of 3D Tiles Next. Most of the features that are shown in these demos are now part of the 3D Tiles 1.1 core specification, or extensions of 3D Tiles 1.1)</sub>
+
+| <div style="width:200px">&nbsp;</div> |  |
+| :--- | :--- |
+| <a href="https://demos.cesium.com/ferry-building">Photogrammetry Classification<br/><img src="figures/ecosystem/CesiumDemoScreenshot-PhotogrammetryClassification.jpg"/></a> | Components of the geometry in 3D Tiles can receive identifiers, and be associated with metadata via these identifiers. In this example, building components in a photogrammetry model are classified based on their type. |
+| <a href="https://demos.cesium.com/owt-uncertainty">Property Textures<br/><img src="figures/ecosystem/CesiumDemoScreenshot-PropertyTextures.jpg"/></a> | High-frequency data can be associated with the surface of geometry, by storing metadata for each surface point (texel) in a property texture |
+| <a href="https://demos.cesium.com/cdb-yemen">Metadata<br/><img src="figures/ecosystem/CesiumDemoScreenshot-Properties.jpg"/></a> | Metadata can be associated with elements of a tileset on different levels of granularity, and queried at runtime to support analytics and styling based on the property values. |
+| <a href="https://demos.cesium.com/owt-globe">S2 Base Globe<br/><img src="figures/ecosystem/CesiumDemoScreenshot-S2BaseGlobe.jpg"/></a> | An extension allows to divide geographic data into [S2 Cells](https://s2geometry.io/) to represent the whole globe with less distortion than traditional mapping approaches, and without singularities at the poles. |
```

Comment:
Make sure to copy over the latest `figures` folder, these ones are missing on this branch

Reply:
Done